CHANGELOG

```diff
@@ -1,6 +1,8 @@
 v3.9.1 (XXXX-XX-XX)
 -------------------
 
+* Added new server option: --icu-language. 
+
```

Comment:
Please add a few words about option meaning, like "Used instead of --default-language to set pure ICU collator"